main.cpp

```diff
@@ -51,21 +51,11 @@ Project 4: Part 4 / 9
  Wait for my code review.
  */
 
-struct Point
-{
-    Point& multiply(float m)
-    {
-        x *= m;
-        y *= m;
-        return *this;
-    }
-private:
-    float x{0}, y{0};
-};
-
-
-
```

Comment:
- [ ] Move `Point` back here.   Use Forward Declaration to make your constructors work.